docker/master/Dockerfile

```diff
@@ -63,9 +60,9 @@ RUN cd /infer && \
 
 FROM debian:stretch-slim AS executor
 
-# Install python 2.7 since infer requires it to run
+# Install python 3.7 since infer Clang plugins requires it to run
```

Comment:
To be pedantic, the plugins only need python for the build I think.

Reply:
Changed this line